### ISSUE TITLE
React 18 compatibility 

### ISSUE THREAD

**Issue by yairyakobson** [AndreasFaust/react-ticker#60]
I believe that the current version (1.3.2) is not compatible with react 18.1

This is what it shows in the console:
PS directory> npm i react-ticker
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: tiktokclone@0.1.0
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"^18.1.0" from the root project

**Comment by Isrealsix** [comment]
Faced same problem, I think for now the only solution is to use react 17 (

**Comment by mariukhin** [comment]
@yair2000 @Isrealsix try to use react@18.0.0 version. It works correct for me.

**Comment by Isrealsix** [comment]
@mariukhin did you use an older version of react-ticker? Or any workaround?

**Comment by yairyakobson** [comment]
Yes, I tried an older version, but from what I understood, each version is compatible with a different version of React

**Comment by yairyakobson** [comment]
> @yair2000 @Isrealsix try to use react@18.0.0 version. It works correct for me.

Tried it. It didn't work

**Comment by Isrealsix** [comment]
@yair2000 run "npm config set legacy-peer-deps true" and install again, it should install perfectly

**Comment by mariukhin** [comment]
@yair2000 I installed react-ticker@1.3.2 version using yarn package manager

**Comment by hongweitang** [comment]
Run your install with the flag `-legacy-peer-deps`. It uses peerDependencies installation behavior from earlier npm versions. Npm v7 and higher installs peerDependencies by default and it causes the issue we have in the case here. 

But I hope we don't need to have this flag soon since a pull requests was issued to fix that.

**Comment by mxsxs2** [comment]
A CI friendly workaround is to add this to you package.json:
```
"overrides": {
        "react-ticker":{
            "react": "^18.0.0",
            "react-dom": "^18.0.0"
        }
    }
```

This overrides the react and react-dom versions for react-ticker.
